app/routes/about-us.tsx

```diff
@@ -1,3 +1,77 @@
+            leberkas ham rump sirloin jerky chicken turkey ground round.
+          </h3>
+          <div className="w-full sm:w-90 sm:items-end">
+            <div className="w-full sm:w-48 text-white right-2.5 bottom-2.5 bg-warm hover:bg-gray-500 focus:ring-2 focus:outline-none focus:ring-blue-500 font-semibold rounded-lg italic text-md px-2 md:px-4 py-4 lg:text-2xl">
+              Join Group
+            </div>
+          </div>
+        </div>
+        {/* <div className="flex justify-center content-center bg-secondary">
+          <h1>TODO: Place holder for event images marquee</h1>
+        </div> */}
+        <div className="max-w-screen-md mx-auto sm:w-4/5 flex flex-col px-8 py-8 items-center justify-center">
+          <div className="w-[320px] flex self-start mx-auto pb-3">
+            <img
+              className="w-full"
+              src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1706805456/target_g3b96r.png"
+              alt="Arrow in targeet"
+            />
+          </div>
+          <div className="pb-3 w-full">
+            <h2 className="sm:w-100 font-bold sm:font-extrabold sm:text-4xl">MISSION</h2>
+          </div>
+          <h3 className="w-50 sm:text-lg sm:w-90 sm:font-semibold pb-6">
+            Shank jerky shoulder short loin corned beef chislic ham cupim. Ribeye andouille jowl, short ribs chislic.
+          </h3>
+        </div>
+        <div className="max-w-screen-md mx-auto sm:w-4/5 flex flex-col px-8 py-8 items-center justify-center">
+          <div className="w-[320px] flex self-start mx-auto pb-3">
+            <img
+              className="w-full"
+              src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1706806762/binoculars_vuhkgz.png"
+              alt="Kid looking through binoculars"
+            />
+          </div>
+          <div className="pb-3 w-full">
+            <h2 className="sm:w-100 font-bold sm:font-extrabold sm:text-4xl">VISION</h2>
+          </div>
+          <h3 className="w-50 sm:text-lg sm:w-90 sm:font-semibold pb-6">
+            Shank jerky shoulder short loin corned beef chislic ham cupim. Ribeye andouille jowl, short ribs chislic.
+          </h3>
+        </div>
+        <div className="max-w-screen-md mx-auto sm:w-4/5 flex flex-col px-8 py-8 items-center justify-center">
+          <div className="w-[320px] flex self-start mx-auto pb-3">
+            <img
+              className="w-full"
+              src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1706806983/crowd_ld0rvy.png"
+              alt="Diversity"
+            />
+          </div>
+          <div className="pb-3 w-full">
+            <h2 className="sm:w-100 font-bold sm:font-extrabold sm:text-4xl">CULTURE</h2>
+          </div>
+          <h3 className="w-50 sm:text-lg sm:w-90 sm:font-semibold pb-6">
+            Shank jerky shoulder short loin corned beef chislic ham cupim. Ribeye andouille jowl, short ribs chislic.
+          </h3>
+        </div>
+      </div>
+    </>
+  );
 }
```

Comment:
need to find a way to credit the creator and flat Icon:

https://support.flaticon.com/s/article/Attribution-How-when-and-where-FI?language=en_US

Reply:
> need to find a way to credit the creator and flat Icon:
> 
> https://support.flaticon.com/s/article/Attribution-How-when-and-where-FI?language=en_US

Let's address this in another PR as it may deal with other pages with photos

Reply:
created a issue to keep track of crediting artist: https://github.com/social-plan-it/plan-it-social-web/issues/94